internal/campaigns/run_steps.go

```diff
@@ -157,6 +182,70 @@ func runSteps(ctx context.Context, wc *WorkspaceCreator, repo *graphql.Repositor
 	return diffOut, err
 }
 
+func parseStepRun(stepCtx StepContext, run string) (*template.Template, error) {
+	t := template.New("step-run").Delims("${{", "}}")
+
+	funcMap := template.FuncMap{
+		"repository_name":    func() string { return stepCtx.Repository.Name },
```

Comment:
I assume this is a function (instead of a string) because the repository name can change? How/when does that happen?

Reply:
It's a function because (a) `FuncMap` only accepts functions and (b) we want to add it to the `FuncMap` so we can access the repository name through a `lower_case` var. I think it looks cleaner than accessing the repository name through the `StepContext`, which you can still do with `${{ .Repository.Name }}`.

It was a design idea we had yesterday and tried out, not set in stone :)